### ISSUE TITLE
bowtie-1.2.2: ebwt.h:865:44: error: cannot convert 'TIndexOffU* const' {aka 'long unsigned int* const'} to 'uint32_t*' {aka 'unsigned int*'} in return

### ISSUE THREAD

**Issue by mmokrejs** [BenLangmead/bowtie#90]
Hi,
  I tried to compile bowtie1 from `bug_fixes` branch as of https://github.com/BenLangmead/bowtie/commit/f253e204f5ba4565d14ae1b300ea6cb10dfe4660 with `gcc-8.2.0` also with `-std=c++03` but it is failing here:

```
x86_64-pc-linux-gnu-g++ -std=c++03 -Wno-unused-local-typedefs -Wno-unused-but-set-variable -Wno-unused-variable -DCOMPILER_OPTIONS="\"-std=c++03 -Wno-unused-local-typedefs -Wno-unused-but-set-variable -Wno-unused-variable -Wl,-O1 -Wl,--as-needed -Wl,--hash-style=both -DWITH_TBB -DPOPCNT_CAPABILITY  \"" -Wl,-O1 -Wl,--as-needed -Wl,--hash-style=both -DWITH_TBB -DPOPCNT_CAPABILITY    \
	-fno-strict-aliasing -DBOWTIE_VERSION="\"`cat VERSION`\"" -DBUILD_HOST="\"`hostname`\"" -DBUILD_TIME="\"`date`\"" -DCOMPILER_VERSION="\"`x86_64-pc-linux-gnu-g++ -v 2>&1 | tail -1`\"" -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -D_GNU_SOURCE  -DPREFETCH_LOCALITY=2 -DBOWTIE_MM -DBOWTIE_SHARED_MEM -DBOWTIE_64BIT_INDEX -DNDEBUG -Wall -Wno-unused-parameter -Wno-reorder -Wno-unused-local-typedefs \
	 -isystem ./SeqAn-1.1 -I third_party -I third_party \
	-o bowtie-build-l ebwt_build.cpp \
	ccnt_lut.cpp ref_read.cpp alphabet.cpp shmem.cpp edit.cpp ebwt.cpp  bowtie_build_main.cpp \
	-Wl,-O1 -Wl,--as-needed  -lz -lpthread -ltbb -ltbbmalloc_proxy 
...
ebwt.h: In member function 'uint32_t* Ebwt<TStr>::isa() const':
ebwt.h:865:44: error: cannot convert 'TIndexOffU* const' {aka 'long unsigned int* const'} to 'uint32_t*' {aka 'unsigned int*'} in return
  uint32_t*   isa() const          { return _isa; } /* check */
                                            ^~~~
```

Here is the full log: [build.log.txt](https://github.com/BenLangmead/bowtie/files/2879574/build.log.txt)

I have SeqAn-1.4 installed system-wide (in case it collides with the bundled 1.1 copy).

**Comment by mmokrejs** [comment]
Ah, I see in commit log https://github.com/BenLangmead/bowtie/commit/58c6ac97b1938909881877ef83167f5eff0e8ab1 on the master branch. Aftery appying it to the `bug_fixes` branch I can compile it with the bundled SeqAn-1.1 sources. Addition of `-std=c++03` did not make a difference.

**Comment by ch4rr0** [comment]
Great! I'll look into updating `bug_fixes` with the changes from `master`.

**Comment by ch4rr0** [comment]
This change was included in our latest release, v1.2.3
